cw/cw305/kmac_cw_demo.py

```diff
@@ -0,0 +1,61 @@
+import argparse
```

Comment:
Please add the OpenTitan license header to all files that you wrote yourself. You can find the license header here: https://docs.opentitan.org/doc/project/detailed_contribution_guide/#writing-code

Reply:
done